lib/src/command/outdated.dart

```diff
@@ -84,7 +95,7 @@ class OutdatedCommand extends PubCommand {
           upgradablePackages = (await resolveVersions(
```

Comment:
We still need to catch the exception thrown if there is a `SolveFailure`.
Can be another CL.
I think I had that almost ready in my local branch